halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/validate/v1/persistentStorage/S3Validator.java

```diff
@@ -16,15 +16,50 @@
 
 package com.netflix.spinnaker.halyard.config.validate.v1.persistentStorage;
 
+import com.amazonaws.SdkClientException;
+import com.amazonaws.auth.AWSCredentialsProvider;
+import com.amazonaws.auth.DefaultAWSCredentialsProviderChain;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.model.Bucket;
+import com.netflix.spinnaker.front50.config.S3Config;
+import com.netflix.spinnaker.front50.config.S3Properties;
 import com.netflix.spinnaker.halyard.config.model.v1.node.Validator;
 import com.netflix.spinnaker.halyard.config.model.v1.persistentStorage.S3PersistentStore;
 import com.netflix.spinnaker.halyard.config.problem.v1.ConfigProblemSetBuilder;
+import com.netflix.spinnaker.halyard.core.problem.v1.Problem;
+import com.netflix.spinnaker.halyard.core.tasks.v1.DaemonTaskHandler;
 import org.springframework.stereotype.Component;
 
+import java.util.Optional;
+
 @Component
 public class S3Validator extends Validator<S3PersistentStore> {
   @Override
   public void validate(ConfigProblemSetBuilder ps, S3PersistentStore n) {
-    // TODO(lwander) find someone to handle https://github.com/spinnaker/halyard/issues/116
+    DaemonTaskHandler.message(String.format("Validating %s with %s", n.getNodeName(), getClass().getSimpleName()));
+    AWSCredentialsProvider credentialsProvider = DefaultAWSCredentialsProviderChain.getInstance();
```

Comment:
It's probably best if you instantiate these credentials by importing front50 as a dependency. It already does things like [create a nonexistent bucket](https://github.com/spinnaker/front50/blob/0c0ed5848b6f46bfc315e976c86d690b4b850758/front50-s3/src/main/java/com/netflix/spinnaker/front50/model/S3StorageService.java#L61) and verify that you have access to it.